api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
<p>crop_weighted_value are doubled due to the duplication as commented above.</p>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>